apps/coscrad-frontend/src/store/slices/ResourcesSlice.ts

```diff
@@ -0,0 +1,69 @@
+import { IAggregateInfo } from '@coscrad/api-interfaces';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { getConfig } from '../../config';
+
+const RESOURCES = 'resources';
+
+export type ResourcesState = {
+    infos: IAggregateInfo[];
+    resources: unknown[];
+    isLoading: boolean;
+};
+
+const initialState: ResourcesState = {
+    // We may want a separate resourceInfoSlice
+    infos: [],
+    resources: [],
+    isLoading: false,
+};
+
+// TODO move to api-interfaces
+type FSA<TPayload = unknown> = {
+    type: string;
+    payload: TPayload;
+};
+
+// we might want to type this and keep it as a helper for testing \ sandboxing
+const delay = async (t, v) => new Promise((resolve) => setTimeout(resolve.bind(null, v), t));
+
+export const fetchResourceInfos = createAsyncThunk(`${RESOURCES}/fetchInfos`, async () => {
+    console.log('initialting requestt!');
+    const baseUrl = getConfig().apiUrl;
+
+    // Should we include "api" here?
+    const endpoint = `${baseUrl}/api/resources`;
```

Comment:
Based on our discussion today - no.  Any reason to hardwire it?

Reply:
Yes. Chicken and the egg problem. This resource info endpoint gives you links that it generates, thereby dynamically telling you what the global prefix is. However, there is no endpoint to give you links to resources. We could make the base route return the global prefix to get the ball rolling. Otherwise, we need a good way to statically share the global prefix between frontend and backend. Note that we can't use real env vars on the frontend.

Reply:
Fixed.